### PR TITLE
feat(Foxford): add presence

### PR DESCRIPTION
**Pull request by cramatsu**
## Description 

Adding presence for **https://foxford.ru/**

Note to the founders of this project:
This online school is the most visited in the Russian Federation (more than `300k` students).
Peace ✌🏿

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Comment by cramatsu** [comment]
Yo, what's wrong with metadata.json
The location and data of the file are the same as the rest

**Comment by DarkVillager** [comment]
> Yo, what's wrong with metadata.json The location and data of the file are the same as the rest

Nothing, just wait for the fix #6800

**Comment by cramatsu** [comment]
> > Yo, what's wrong with metadata.json The location and data of the file are the same as the rest
> 
> 
> 
> Nothing, just wait for the fix #6800

I'll be looking forward to it🥰

File: websites/F/Foxford/presence.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "f1024",
+	};
+
+	switch (true) {
+		case location.pathname.split("/").pop() in routes:
+			presenceData.details =
+				routes[
+					location.pathname.split("/").pop() as unknown as keyof typeof routes
```
Comment (EncryptedDev) [inline websites/F/Foxford/presence.ts:29]: I don't think the unknown type needs to be here

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/F/Foxford/presence.ts (hunk 1 above)
Comment (cramatsu) [reply websites/F/Foxford/presence.ts:29]: Without type extension, the compiler starts to worry that you are doing something wrong

**Review by Bas950 — APPROVED** [review]

File: websites/F/Foxford/presence.ts (hunk 1 above)
Comment (cramatsu) [reply websites/F/Foxford/presence.ts:29]: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**